images/linux/scripts/installers/homebrew.sh

```diff
@@ -11,14 +11,13 @@ source $HELPER_SCRIPTS/install.sh
 
 # Install the Homebrew on Linux
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
-eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)
+brew_shellenv="/home/linuxbrew/.linuxbrew/bin/brew shellenv"
 
-# Update /etc/environemnt
-## Put HOMEBREW_* variables
-## The variable `HOMEBREW_SHELLENV_PREFIX` will be exported to avoid adding duplicate entries to the environment variables.
-HOMEBREW_SHELLENV_PREFIX='' brew shellenv | grep 'export HOMEBREW'| sed -E 's/^export (.*);$/\1/' | sudo tee -a /etc/environment
+# Update /etc/environment
+## Put HOMEBREW_* variables.
+$brew_shellenv | grep 'export HOMEBREW' | sed -E 's/^export (.*);$/\1/' | sudo tee -a /etc/environment
```

Comment:
This is going to prevent `MANPATH` and `INFOPATH` from being set correctly.

Reply:
> This is going to prevent `MANPATH` and `INFOPATH` from being set correctly.

Do we really need these vars in non-interactive CI?

Reply:
Perhaps not. Feels perhaps worth adding for completeness and having a way to handle this if Homebrew needs additional variables in future.